test/truefi2/TrueCreditAgency.test.ts

```diff
@@ -187,4 +205,82 @@ describe('TrueCreditAgency', () => {
       expect(await tusd.balanceOf(tusdPool.address)).to.equal(parseEth(1e7))
     })
   })
+
+  describe('Interest calculation', () => {
+    beforeEach(async () => {
+      await creditAgency.allowBorrower(borrower.address, true)
+      await creditAgency.allowBorrower(owner.address, true)
+      await creditAgency.setRiskPremium(1000)
+      await creditOracle.setScore(owner.address, 255)
+    })
+
+    it('interest for single borrower and stable rate', async () => {
+      await creditAgency.connect(borrower).borrow(tusdPool.address, 1000)
+      await timeTravel(YEAR)
```

Comment:
Should we do a year-long timeTravel, while we expect the interest to be paid off monthly?

Reply:
I don't see any connection between these things. Year-long time travel makes tests much easier to understand as we don't have to divide by 12